index.js

```diff
@@ -24,21 +24,55 @@ MIT License
 		}));
 	};
 
+	// Function to check if the given device is a phone or ipad
+	function isPhone() {
+		return /android|ipad/i.test(navigator.userAgent);
+	}
+
+	// Check how long its takes for the given code to execute
+	// It return a value after execution
+	// If value is greater than 60 then dev tool is open
+	// If value is less than 60 then dev tool is not open
```

Comment:
This sounds very fragile. Is this really reliable enough? The performance difference between a low-end Android phone and the latest iPad is extreme.